src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -440,6 +442,7 @@ class VAMap extends Component {
           }
           this.props.fetchVAFacility(r.id, r);
         },
+        markerText: markers.next().value,
```

Comment:
This is really cool and looks great!

My only note would be is if you're concerned about this being synchronized with the markers in the ResultList. If it's possible in some way for the two to fall out of sync, then it may be worth binding the marker letters to each result through the reducer before the result list enters state. So this line would just be `markerText: r.markerText`. But that may not be necessary.